src/components/MessageList.vue

```diff
@@ -71,6 +71,7 @@
 
 <script>
 
+import Logger from '@/libs/Logger';
```

Comment:
Just to be consistent, this should be moved down a few lines to the other local imports. ie. after the strftime module import line